src/content/updates/whats-new/whats-new.mdx

```diff
@@ -1,28 +1,223 @@
 ---
-title: What’s new
+title: What's new
 ---
 
-## February 2019
-
-The Carbon X launch is just around the corner, and we've officially entered Beta! That means the core team has been hard at work ensuring the new system is ready to ship. We have tons of improvements headed your way, including implementation of the new IBM Design Language and an all-new website.
-
-We expect X to launch mid-March, so keep your eyes peeled. If you're interested in getting started before our public launch, we're always looking for beta testers to help us squash bugs and improve the Carbon Design System.
-
+## July 2019
+
+[**Changelog: v10.4.0**](https://github.com/carbon-design-system/carbon/releases/tag/v10.4.0#changelog)
+
+Our v10.4.0 release offers a wide range of enhancements and improvements in the following areas:
+
+- Theming enhancements
+- Icon updates
+- UI shell enhancements
+- Support and component enhancements
+- Component specification
+- New contribution guidelines
+
+We also introduced general updates to our website and have an ongoing support project for managing issue triage and fixing bugs.
+
+### Theming enhancements
+
+#### Enhancements 🛠 
+
+- Added color theme dropdown in vanilla development environment (https://the-carbon-components.netlify.com) https://github.com/carbon-design-system/carbon/pull/3073
```

Comment:
Can we change this link to be in the text rather than the whole url? like `[link word here](url)`. Can use the PR number like `(#3073)[url]` at the end of the bullet point.

Reply:
👍